index.js

```diff
@@ -0,0 +1,330 @@
+                stopPlay(cellArray);
+                currentPlayer = 'o';
+                status.textContent = `${capitalize(playerTwo)}'s turn!`
+                let randomCell = cellArray[Math.floor(Math.random() * 9)];
+                while (randomCell.textContent !== '' && randomCell) {
+                    randomCell = cellArray[Math.floor(Math.random() * 9)];
+                }
+                randomCell.removeEventListener('click', alreadyClicked)
+                randomCell.addEventListener('click', clicked);
+                setTimeout(() => {
+                    randomCell.click()
+                }, 1000);
+                cellArray.forEach(function (cell) {
+                    cell.addEventListener('click', clicked);
+                })
+            } else {
+                currentPlayer = 'x';
+                status.textContent = `${capitalize(playerOne)}'s turn!`;
+                cellArray.forEach(function (cell) {
+                    cell.addEventListener('click', alreadyClicked);
+                })
+            }
+        } else if (gameMode === 'twoPlayer') {
+            if (currentPlayer === 'x') {
+                currentPlayer = 'o';
+                status.textContent = `${capitalize(playerTwo)}'s turn!`
+            } else {
+                currentPlayer = 'x';
+                status.textContent = `${capitalize(playerOne)}'s turn!`;
+            }
+        }
+    }
+}
+
+// function that tells the player to select an empty cell if they've selected one that's 
+// already in use
+function alreadyClicked() {
+    if (gameMode === 'onePlayer' && currentPlayer === 'o') {
+        let randomCell = cellArray[Math.floor(Math.random() * cellArray.length)];
+        randomCell.click();
+    }
+    status.textContent = 'Please select an empty cell.';
+}
+
+startBtn.addEventListener('click', () => {
+    start();
+    onePlayer.disabled = true;
+    twoPlayer.disabled = true;
+})
+
+// function to remove event listener on cells after someone has won
+function stopPlay(cellArray) {
+    cellArray.forEach(function (cell) {
+        cell.removeEventListener('click', clicked);
+        cell.removeEventListener('click', alreadyClicked);
+    })
+}
+
+// function to check whether or not the player has won
+function checkWin() {
+    for (let combo of Object.values(WINNING_COMBOS)) {
+        if (combo[0].textContent === '') { } else if (
+            combo[0].textContent === combo[1].textContent &&
+            combo[0].textContent === combo[2].textContent
+        ) {
+            markWinner(combo);
+            status.textContent = `${capitalize(currentPlayer)} wins!`;
+            startBtn.disabled = false;
+            stopPlay(cellArray);
+            return true;
+        }
+    }
+}
+
+// function that highlights the winning squares 
+function markWinner(combo) {
+    for (let cell of combo) {
+        cell.style.backgroundColor = '#a1f4f3';
+        cell.style.border = '3px solid #42aff7';
+    }
+}
```

Comment:
I believe this isn't working correctly because when you add your on hover effects, you're creating an inline style already that would be conflicting with this.

Reply:
The hover effects were a mistake...

Reply:
I realize that they could have been done a lot better

Reply:
That's why, in the "final" version we pushed, it would activate only when the user clicked the top left square, as sort of an "easter egg"... I did not realize that this would effect gameplay and obscure the visibility of the mark winner function.  I was meaning to re-do them in CSS or else take it out completely. Oh well. Guess I was only thinking of the end user experience on this and not how a coder would read the program.